server/src/main/java/org/apache/iotdb/db/mpp/sql/parser/ASTVisitor.java

```diff
@@ -1564,6 +1564,11 @@ private String[] parsePrivilege(IoTDBSqlParser.PrivilegesContext ctx) {
     return privileges.toArray(new String[0]);
   }
 
+  @Override
+  public Statement visitSetStorageGroup(IoTDBSqlParser.SetStorageGroupContext ctx) {
+    return super.visitSetStorageGroup(ctx);
```

Comment:
You should construct SetStorageGroupStatement here by yourselft. You can refer the implementation in `visitSetStorageGroup` of `IoTDBSqlVisitor`.